Lib/profiling/sampling/live_collector.py

```diff
@@ -0,0 +1,2005 @@
+            width: Available width
+
+        Returns:
+            Next available line number
+        """
+        A_DIM = self.display.get_attr("A_DIM")
+        A_BOLD = self.display.get_attr("A_BOLD")
+
+        # Legend line
+        legend = "nsamples: direct/cumulative (direct=executing, cumulative=on stack)"
+        self.add_str(line, 0, legend[: width - 1], A_DIM)
+        line += 1
+
+        # Controls line with status
+        sort_names = {
+            "tottime": "Total Time",
+            "nsamples": "Direct Samples",
+            "cumtime": "Cumulative Time",
+            "sample_pct": "Sample %",
+            "cumul_pct": "Cumulative %",
+        }
+        sort_display = sort_names.get(
+            self.collector.sort_by, self.collector.sort_by
+        )
+
+        # Build status indicators
+        status = []
+        if self.collector.finished:
+            status.append("[PROFILING FINISHED - Press 'q' to quit]")
+        elif self.collector.paused:
+            status.append("[PAUSED]")
+        if self.collector.filter_pattern:
+            status.append(f"[Filter: {self.collector.filter_pattern}]")
+        status_str = " ".join(status) + " " if status else ""
+
+        if self.collector.finished:
+            footer = f"{status_str}"
+        else:
+            footer = f"{status_str}Sort: {sort_display} | 'h':help 'q':quit"
+        self.add_str(
+            line,
+            0,
+            footer[: width - 1],
+            A_BOLD
+            if (self.collector.paused or self.collector.finished)
+            else A_DIM,
+        )
+
+        return line + 1
+
+    def render_filter_input_prompt(self, line, width):
+        """Draw the filter input prompt at the bottom of the screen."""
+        A_BOLD = self.display.get_attr("A_BOLD")
+        A_REVERSE = self.display.get_attr("A_REVERSE")
+
+        # Draw prompt on last line
+        prompt = f"Function filter: {self.collector.filter_input_buffer}_"
+        self.add_str(line, 0, prompt[: width - 1], A_REVERSE | A_BOLD)
+
+
+class HelpWidget(Widget):
+    """Widget for rendering the help screen overlay."""
+
+    def render(self, line, width, **kwargs):
+        """
+        Render the help screen.
+
+        Args:
+            line: Starting line number (ignored, help is centered)
+            width: Available width
+            kwargs: Must contain 'height' key
+
+        Returns:
+            Next available line number (not used for overlays)
+        """
+        height = kwargs["height"]
+        A_BOLD = self.display.get_attr("A_BOLD")
+        A_NORMAL = self.display.get_attr("A_NORMAL")
+
+        help_lines = [
+            ("Tachyon Profiler - Interactive Commands", A_BOLD),
+            ("", A_NORMAL),
+            ("Navigation & Display:", A_BOLD),
+            ("  s           - Cycle through sort modes", A_NORMAL),
+            ("  +           - Faster refresh rate", A_NORMAL),
+            ("  -           - Slower refresh rate", A_NORMAL),
```

Comment:
Might be useful to avoid confusion that this isn't the sampling interval.

```suggestion
            ("  +           - Faster display refresh rate", A_NORMAL),
            ("  -           - Slower display refresh rate", A_NORMAL),
```